doc/content/getting-started/aws/ecs/architecture/aws-databases.drawio

```diff
@@ -1 +1 @@
-<mxfile host="Electron" modified="2020-09-17T13:04:21.503Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/13.6.2 Chrome/83.0.4103.122 Electron/9.2.0 Safari/537.36" etag="J-RRurA1U_-Mx2vtPfWx" version="13.6.2" type="device"><diagram id="EGYVQjSTU_BKqKpe2UHH" name="Page-1">7Vltb9owEP41fCzKGwQ+lgDbJCZ1Q1OlfqlM4iTWnDhznAL79TsnDpA4rJtGO7qFIuF7zjmf754zvjKwvWT3jqMs/sgCTAeWEewG9nxgWaZpjOFDInuFGIZdIREngcKOwJp8x/VEhRYkwHljomCMCpI1QZ+lKfZFA0Ocs21zWshoc9UMRVgD1j6iOnpPAhHXGxtPj4r3mESxWnpiuZUiQfVktZM8RgHbnkD2YmB7nDFRjZKdh6mMXh2X6rnlGe3BMY5T8SsPhIZnrb6QLEnn91Y49b89ONmNsvKEaKE2fPuECEUbQonYg+aBpVj5L/Z1UEJCqcco4yCmcoI9ywVnX3ENDizbdNzF7BY0AcpjLH0wQXjCXBCI7i0lUQqYYBmgIUvFWpk3lNxlSjkLNvDubBTMQ2yBlZglWHC5EfWAYwxHioH7GqnE7TG9I3M6dFQe45PkHuYixaroYP8YeBio2P9GHmwtD3ecPCGBZeCLTYqFloSMkVSUfoxm8AbPPGMwAo0npaE1agFt2W0Cpi5JG02gLbtNwGybN1vrm20HTwBNapg3WusbJw7C256xQlCSAgPrc0ASKeIoIPhIJkXWWCRUEXIbE4HXGfJlVLdwiNV0rE4i0+qgJ1RyJsfJLpKn3hBtc2cYcVZk5ZIf4Czq1D7m2C841NXjcfK6rJva8NkiOq04wBfjpbWcnK0npCSKQyHNwvZIGq1KaW7/rMIOxWpcptxco1Frrl5rztjQC82cvFShmSP9xCs442ggj88xlfHacBhFcnTHchFxvP600nUco+BmC+nUz8dOLrZCDn9L6bfGUdA5cxeUJ7o54WCIlLxKGZexa1MCXs5s2kWisHy1k1szZ4U2mMI+iTK/YUKw5Flq+eAV5s1aeq5uUJ5V4QjJTvrRXUgc56zgPq7KCMKcdxUUqpJ2EZLCpaTBUtPUaTrpYOnLkdTVSLqgKBfEQ36MO5n6GQck70l6XSTFZdL8MmkXYao1vjamTnum9kztumdbV8bU2p//qtWpm1GVBNu4hl7H0pvOvtnpm5233+yMp0PXaRac/dcbHvn8BRseltJ9/wX9tvudqc7Tjsb8db+d9X+A/ck9smfpP3KNtEZXyFWn52rP1Y77tvmaXAXx+NtRqTv5Cc5e/AA=</diagram></mxfile>
```

Comment:
If Draw.io doesn't automatically insert the newline, I think we should not enforce it.

So add to the `.editorconfig`:

```
[*.drawio]
insert_final_newline = false
```